widget/ui/src/components/Collapse/Collapse.styles.tsx

```diff
@@ -0,0 +1,18 @@
+import { styled } from '../../theme';
+
+export const Container = styled('div', {
+  padding: '$15',
+  paddingBottom: '$5',
+  borderRadius: '$xm',
+  cursor: 'pointer',
+  backgroundColor: '$neutral200',
+  textAlign: 'center',
+  '&:hover': {
+    backgroundColor: '$surface600',
+  },
```

Comment:
For general use case it is better to remove the icon and style from the collapse